apps/redi-connect/src/pages/app/me/Me.tsx

```diff
@@ -34,6 +35,23 @@ import { useLoading } from '../../../hooks/WithLoading'
 import './Me.scss'
 import OnboardingSteps from './OnboardingSteps'
 
+// We controll the stepper by passing the current step index (zero-based) as the activeStep prop
```

Comment:
Can we move it to below the `<Me>` component? That's the main point of interest for people opening this file. This is "just" a helper function.